static/docs/use-cases/data-storage-on-external-drive.md

```diff
@@ -0,0 +1,207 @@
+# Data Storage On External Hard Drive
```

Comment:
Maybe just "...on External Drives"? Not sure. What do you think @shcheklein?

p.s. please make "on" lower case for consistency with other document titles (e.g. see https://dvc.org/doc/get-started/add-files).